crawlers/mooncrawl/mooncrawl/stats_worker/queries.py

```diff
@@ -118,12 +124,12 @@ def data_generate(
                     ],
                 }
             ).encode("utf-8")
-            push_statistics(
-                s3=s3,
-                data=data,
-                key=f"{MOONSTREAM_S3_QUERIES_BUCKET_PREFIX}/queries/{query_id}/data.{file_type}",
-                bucket=bucket,
-            )
+        push_data_to_bucket(
+            data=data,
+            key=key,

```

Comment:
Where is this key generated?

Reply:
Any Api call level once for presignUrl and bucket push